frontend/src/app/containers/App.js

```diff
@@ -59,17 +59,16 @@ function subscribeToBasket(email, callback) {
 }
 
 export default connect(
-  state => ({
+  (state, ownProps) => ({
     addon: state.addon,
+    experiment: getExperimentBySlug(state.experiments, ownProps.params.slug),
```

Comment:
I'm kind of leary of this solution. Right now, `/experiment/` is the only page that uses `:slug` in the router. But, that might not always be true and this lookup is specific to the experiment details page. This has us looking up the experiment for *every* page, whether or not it needs one or has a slug.

I've been wondering if maybe a cleaner solution would be to put a wrapper around the `ExperimentPage` component in the router that does the experiment lookup & passes it in as a prop. That way, the lookup logic would be limited to the router & the page.